slick/src/main/scala/geotrellis/slick/PostGisProjectionSupport.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2016 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.slick
+
+import geotrellis.vector._
+
+import slick.ast.FieldSymbol
+import slick.driver.{JdbcDriver, PostgresDriver}
+import slick.jdbc.{PositionedParameters, PositionedResult, SetParameter}
+import com.github.tminglei.slickpg.geom.PgPostGISExtensions
+
+import java.sql._
+import scala.reflect.ClassTag
+
+/**
+ * This class provides column types and extension methods to work with Geometry columns
+ *  associated with an SRID in PostGIS.
+ *
+ * Sample Usage:
+ * <code>
+ * val PostGIS = new PostGisProjectionSupport(PostgresDriver)
+ * import PostGIS._
+ *
+ * class City(tag: Tag) extends Table[(Int,String,Projected[Point])](tag, "cities") {
+ *   def id = column[Int]("id", O.PrimaryKey, O.AutoInc)
+ *   def name = column[String]("name")
+ *   def geom = column[Projected[Point]]("geom")
+ *   def * = (id, name, geom)
+ * }
+ * </code>
+ *
+ * based on [[package com.github.tminglei.slickpg.PgPostGISSupport]]
+ */
+trait PostGisProjectionSupport extends PgPostGISExtensions { driver: PostgresDriver =>
```

Comment:
Bad rebase, slick is back.

Reply:
I'm not sure how that keeps happening...

Reply:
@jbouffard btw it is not `slippy` :D

Reply:
@pomadchin Thanks for the reminder haha